cli_tools/gce_ovf_import/README.md

```diff
@@ -61,7 +61,11 @@ Exactly one of these must be specified:
 + `-no-restart-on-failure` the instance will not be restarted if it’s terminated by Compute Engine.
   This does not affect terminations performed by the user.
 + `-os=OS` Specifies the OS of the image being imported. Execute the tool with `-help` to
-  see the list of currently-supported operating systems.
+  see the list of currently-supported operating systems. When omitted, the tool detects the OS.
++ `-byol` Import using an [existing license](https://cloud.google.com/compute/docs/nodes/bringing-your-own-licenses). These are equivalent:
+  * `-os=rhel-8 -byol`
+  * `-os=rhel-8-byol -byol`
+  * `-os=rhel-8-byol`
```

Comment:
nit: move -byol flag ahead of -os (or add more spaces) to make it more readable (`-os=rhel-8 -byol` looks very similar to `-os=rhel-8-byol`)

Reply:
done!